modules/deleting-the-cluster-logging-operator-by-using-the-cli.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+// * uninstalling/uninstalling-logging.adoc
+
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-07-05
+:_mod-docs-content-type: PROCEDURE
+
+[id="deleting-the-cluster-logging-operator-by-using-the-cli_{context}"]
+= Deleting the {clo} by using the CLI
+
+Delete the {clo} by using the {oc-first}.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You installed the {oc-first}.
+
+.Procedure
+
+. Delete the `ClusterLogForwarder` custom resource (CR) by running the following command:
++
+[source,terminal]
+----
+$ oc delete ClusterLogForwarder instance -n openshift-logging
+----
++
+Change `instance` to the name of your `ClusterLogForwarder` CR if you have used a different name.
+
+. Delete the service account by running the following commands:
++
+[source,terminal]
+----
+$ oc adm policy remove-cluster-role-from-user lokistack-tenant-logs -z logging-collector -n openshift-logging
+----
++
+[source,terminal]
+----
+$ oc adm policy remove-cluster-role-from-user collect-application-logs -z logging-collector -n openshift-logging
+----
++
+[source,terminal]
+----
+$ oc adm policy remove-cluster-role-from-user collect-infrastructure-logs -z logging-collector -n openshift-logging
+----
++
+[source,terminal]
+----
+$ oc adm policy remove-cluster-role-from-user collect-audit-logs -z logging-collector -n openshift-logging
+----
++
+[source,terminal]
+----
+$ oc adm policy remove-cluster-role-from-user logging-collector-logs-writer -z logging-collector -n openshift-logging
+----
++
+[source,terminal]
+----
+$ oc delete serviceaccount logging-collector -n openshift-logging
+----
+
+. Delete the `LogFileMetricExporter` resource if you have deployed it by running the following command:
++
+[source,terminal]
+----
+$  oc -n openshift-logging delete LogFileMetricExporter --all
+----
+
+. Delete the subscription associated with the {clo} by running the following command:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc delete subscriptions.operators.coreos.com -n openshift-logging
```

Comment:
The name of the subscription is missed. 
Besides, the CSV in the `openshift-logging` namespace should be removed, otherwise the operator pod won't be removed.

Reply:
done